src/go/rpk/pkg/config/config.go

```diff
@@ -87,6 +87,7 @@ func defaultMap() map[string]interface{} {
 	var defaultListeners []interface{} = []interface{}{defaultListener}
 	return map[string]interface{}{
 		"config_file": "/etc/redpanda/redpanda.yaml",
+		"pandaproxy":  Pandaproxy{},
```

Comment:
Should we make all these concrete types instead of maps? (not in this PR of course)

Reply:
I think it's easier to read and more type-safe as a concrete type, so yeah. Then this function would be replaced with `toMap(Default())`

Reply:
I created a ticket #1067 to track this.